### PR TITLE
fix: don't show empty errored chain names

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
## Description

fix - don't show error/warning banner about not being able to fetch a chain if the flag is off and we don't have the chain name

we often have issues with chains shortly after we launch them

case in point - bnb node currently sad, and we're showing the error in screenshot below with an empty name, which isn't useful to the user

this is because of the plugin model and we don't have the chain adapter available, hence no chain name

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

n/a

## Risk

basically zero - hiding a banner if we don't have useful info to display

### Engineering

ensure my early return is sane lmeow

### Operations

unable to be tested by ops

## Screenshots (if applicable)

prod currently

image

this diff

image

### CODE REVIEW

**Review by gomesalexandre — APPROVED** [review]
Scrutinized carefully and confirm with MDN `Array.prototype.length`represents the number of elements in that array. The value is an unsigned, 32-bit integer that is always numerically greater than the highest index in the array.

TIL
